backend/migrations/versions/5cdcf6cc1d73_create_and_backfill_dataset_table.py

```diff
@@ -0,0 +1,380 @@
+        bucketPolicyCreated = Column(Boolean, default=False)
+        businessOwnerEmail = Column(String, nullable=True)
+        businessOwnerDelegationEmails = Column(ARRAY(String), nullable=True)
+        stewards = Column(String, nullable=True)
+
+        SamlAdminGroupName = Column(String, nullable=True)
+
+        importedS3Bucket = Column(Boolean, default=False)
+        importedGlueDatabase = Column(Boolean, default=False)
+        importedKmsKey = Column(Boolean, default=False)
+        importedAdminRole = Column(Boolean, default=False)
+        imported = Column(Boolean, default=False)
+
+        autoApprovalEnabled = Column(Boolean, default=False)
+
+    # Create columns in s3_dataset table
+    op.add_column(
+        's3_dataset',
+        sa.Column('label', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('name', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('owner', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('created', sa.DateTime(), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('updated', sa.DateTime(), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('deleted', sa.DateTime(), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('description', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('environmentUri', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('organizationUri', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('region', sa.String(), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('AwsAccountId', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('language', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('topics', postgresql.ARRAY(sa.String()), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('confidentiality', sa.String(), nullable=False, server_default=''),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('tags', postgresql.ARRAY(sa.String()), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('businessOwnerEmail', sa.String(), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('businessOwnerDelegationEmails', postgresql.ARRAY(sa.String()), nullable=True),
+    )
+    op.add_column(
+        's3_dataset',
+        sa.Column('stewards', postgresql.ARRAY(sa.String()), nullable=True),
```

Comment:
We add stewards as string column in upgrade to `dataset` but add it back to `s3_dataset` as Array